src/user_interface/home_page.py

```diff
@@ -1,18 +1,20 @@
 import sys
-
+import os
 from PyQt5.QtCore import pyqtSlot, Qt
 from PyQt5.QtWidgets import QApplication, QWidget, QLabel, QPushButton, QVBoxLayout, QHBoxLayout, QSpacerItem, \
     QSizePolicy, QFileDialog
 
 
 class HomePage(QWidget):
 
+    MIDI_FILE_PATH = "MIDI_FILES" # Path to check for midi files to display. For now just checks folder in same directory
```

Comment:
File directories in Linux are case-sensitive, this should be `MIDI_Files`.

Reply:
i was so close.....